### ISSUE TITLE
Question:`.WEBP` image not used when rendering webpage?..

### ISSUE THREAD

**Issue by gevuong** [Niels-IO/next-image-export-optimizer#227]
Hi, I have a NextJS app with App Router deployed to Github Pages, and this is one of the pages:
https://gevuong.github.io/minimal-i18n-with-app-router/en/our-story

I noticed the optimized `.WEBP` image is not being used when the page is rendered. Instead it's using the `.png` version.
<img width="965" alt="Screenshot 2024-07-05 at 7 17 59 PM" src="https://github.com/Niels-IO/next-image-export-optimizer/assets/27314227/2d0576ec-be64-44b0-b878-a5bd00fd92b6">

In the Network tab, I see that the `.WEBP` version of the image is fetched, but it's a 10x7 image, which is extremely small.

<img width="1546" alt="Screenshot 2024-07-05 at 7 14 48 PM" src="https://github.com/Niels-IO/next-image-export-optimizer/assets/27314227/72c90e23-7bd7-4ad9-bbd5-3e87600f261d">

This is how I'm dynamically importing the image: https://github.com/gevuong/minimal-i18n-with-app-router/blob/2eb7c95e73e4d1537a21b81513e999e3c02d7249/app/%5Blang%5D/our-story/page.tsx#L19

And this is my `next.config.mjs`: https://github.com/gevuong/minimal-i18n-with-app-router/blob/main/next.config.mjs

Question:
1. Any idea why the optimized `.WEBP` image is not being used when the page is rendered?
2. Why is only the smallest WEBP image being fetched and not a larger WEBP image to better fit current screen size?

Any assistance would be greatly appreciated. Thanks!

**Comment by gevuong** [comment]
Update:

I have added more content to https://gevuong.github.io/minimal-i18n-with-app-router/en/our-story since posting this question, but the result remains the same.. 
The `.WEBP` images return a 404, and the original `.png` images are rendered instead..

<img width="1375" alt="Screenshot 2024-07-07 at 10 52 52 PM" src="https://github.com/Niels-IO/next-image-export-optimizer/assets/27314227/162cc12a-648d-4532-aa64-9d6ad642620a">
I suspect that the `basePath` needs to somehow be prepended to `/nextImageExportOptimizer` in the request url like so: https://gevuong.github.io/minimal-i18n-with-app-router/nextImageExportOptimizer/mask_header.48e26024-opt-10.WEBP because when I do so, albeit is extremely small, the `.WEBP` image successfully loads on screen.

I may try updating `nextImageExportOptimizer_exportFolderName` to `/minimal-i18n-with-app-router/nextImageExportOptimizer` instead of just `/nextImageExportOptimizer` to see if that will properly prepend the `basePath` to the request url..




Based on the request url for the `.WEBP` images, it would seem like I have to dynamically import the images and prepend the `basePath` to every image. Is that correct?

I am statically importing these images like so, similar to the recommended static import example shown in the [documentation](https://github.com/Niels-IO/next-image-export-optimizer?tab=readme-ov-file#basic-configuration).

Under `Build with Next.js` the package does show the images are being optimized:
https://github.com/gevuong/minimal-i18n-with-app-router/actions/runs/9834082649/job/27145301495

**Comment by gevuong** [comment]
Update 2:

I figured out how to fix the 404 error. I needed to add a `basePath` prop to the `ExportedImage` component.. 👍🏼 

<img width="1380" alt="Screenshot 2024-07-07 at 11 57 01 PM" src="https://github.com/Niels-IO/next-image-export-optimizer/assets/27314227/f349e009-7ca9-44aa-a2e2-9799f27ee065">

\
\
Which leaves me to my last question:
Why is only the smallest `.WEBP` image being fetched and not a larger `.WEBP` image  (ie. `...opt-1920.WEBP` to better fit current screen size, and why is the `.png` file still being rendered on the web page and not the optimized `.WEBP` image?

When I run the static export files locally, I notice that the optimized `...opt-1920.WEBP` image is being rendered, not the `.png` file, as shown below:
<img width="1078" alt="Screenshot 2024-07-07 at 11 58 28 PM" src="https://github.com/Niels-IO/next-image-export-optimizer/assets/27314227/364ae758-8713-4e63-9c76-50164bb0cc75">

Any help or suggestions would be greatly appreciated! Thanks.
